app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= link_to category_transacs_path(category), class: 'category-transac-link' do %>
+  <div class="category-container">
+    <div class="icon-container">
+      <%#= image_tag(category.icon, class: 'category-img', alt: 'Category image') %>
```

Comment:
I suggest you remove every commented piece of code as it makes your codebase messy and gives it a less professional look😉.